src/integrationTest/java/org/openstreetmap/atlas/mutator/AbstractAtlasMutatorIntegrationTest.java

```diff
@@ -52,9 +52,12 @@
     public static final String OUTPUT_BOUNDARIES_META = OUTPUT + "/" + BOUNDARY_META_NAME;
     public static final String OUTPUT_COUNTRY_LIST = OUTPUT + "/" + AtlasMutator.COUNTRY_AND_LEVELS;
 
-    public static void addResource(final String path, final String name)
+    public static void addAtlasResource(final String path, final Atlas atlas)
     {
-        addResource(path, name, false);
+        final long size = 8192;
```

Comment:
I like to do
```java
final long size = 1024 * 8;
```
Makes it clearer that this number is supposed to be 8 KiB. But this is a minor point.

Reply:
This was already there, must have been shuffled around, but I will fix that.